link_linux.go

```diff
@@ -229,6 +229,21 @@ func (h *Handle) macvlanMACAddrChange(link Link, addrs []net.HardwareAddr, mode
 	return err
 }
 
+
+// LinkSetHardwareAddr sets the hardware address of the link device.
+// Equivalent to: `ip link set $link address $hwaddr`
+func BridgeSetGroupFwdMask(link Link, mask uint16) error {
+	return pkgHandle.BridgeSetGroupFwdMask(link, mask)
+}
+
+// LinkSetHardwareAddr sets the hardware address of the link device.
+// Equivalent to: `ip link set $link address $hwaddr`
+func (h *Handle) BridgeSetGroupFwdMask(link Link, mask uint16) error {
+	bridge := link.(*Bridge)
```

Comment:
what if it is not of type Bridge ?

Reply:
That's a good question.  Note that I just copied/pasted the MulticastSnooping functions.  So those don't have a check for that either.  If I add one here, should it also be added to that function?